packages/schemas/src/barcodes/helper.ts

```diff
@@ -133,11 +133,20 @@ export const createBarCode = async (arg: {
   backgroundColor?: string;
   barColor?: string;
   textColor?: string;
+  includetext?: boolean;
 }): Promise<Buffer> => {
-  const { type, input, width, height, backgroundColor, barColor, textColor } = arg;
+  const {
+    type,
+    input,
+    width,
+    height,
+    backgroundColor,
+    barColor,
+    textColor,
+    includetext = DEFAULT_BARCODE_INCLUDETEXT,
```

Comment:
We need to insert a default value here, as it would change the behavior of existing templates.